docs/user-guide/job/expansions/README.md

```diff
@@ -0,0 +1,194 @@
+---
```

Comment:
In general I think the URLs should not be README.md; the conventions you're using here are incompatible with GitHub's markdown viewer that shows up while rooting around in repos.  /docs/user-guide/job/expansions/index.md is recommended.